config/tables.go

```diff
@@ -14,82 +14,47 @@ type (
 
 	//LogTableCfg contains the configuration for logging
 	LogTableCfg struct {
-		RitaLogTable string
+		RitaLogTable string `default:"logs"`
 	}
 
 	//StructureTableCfg contains the names of the base level collections
 	StructureTableCfg struct {
-		ConnTable         string
-		HTTPTable         string
-		DNSTable          string
-		UniqueConnTable   string
-		HostTable         string
-		IPv4Table         string
-		IPv6Table         string
-		FrequentConnTable string
+		ConnTable         string `default:"conn"`
+		HTTPTable         string `default:"http"`
+		DNSTable          string `default:"dns"`
+		UniqueConnTable   string `default:"uconn"`
+		HostTable         string `default:"host"`
+		IPv4Table         string `default:"ipv4"`
+		IPv6Table         string `default:"ipv6"`
+		FrequentConnTable string `default:"freqConn"`
 	}
 
 	//BlacklistedTableCfg is used to control the blacklisted analysis module
 	BlacklistedTableCfg struct {
-		BlacklistDatabase string
-		SourceIPsTable    string
-		DestIPsTable      string
-		HostnamesTable    string
+		SourceIPsTable string `default:"blSourceIPs"`
+		DestIPsTable   string `default:"blDestIPs"`
+		HostnamesTable string `default:"blHostnames"`
 	}
 
 	//DNSTableCfg is used to control the dns analysis module
 	DNSTableCfg struct {
-		ExplodedDNSTable string
-		HostnamesTable   string
+		ExplodedDNSTable string `default:"explodedDns"`
+		HostnamesTable   string `default:"hostnames"`
 	}
 
 	//BeaconTableCfg is used to control the beaconing analysis module
 	BeaconTableCfg struct {
-		BeaconTable string
+		BeaconTable string `default:"beacon"`
 	}
 
 	//UserAgentTableCfg is used to control the useragent analysis module
 	UserAgentTableCfg struct {
-		UserAgentTable string
+		UserAgentTable string `default:"useragent"`
 	}
 
 	//MetaTableCfg contains the meta db collection names
 	MetaTableCfg struct {
-		FilesTable     string
-		DatabasesTable string
+		FilesTable     string `default:"files"`
+		DatabasesTable string `default:"databases"`
 	}
 )
```

Comment:
The entire initialization function below is now replaced with the `default` tags and a single function call in the config.go file.